pkg/acirenderer/acirenderer.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2015 CoreOS, Inc.
```

Comment:
Could you please drop this? We aren't licensing appc/spec as a CoreOS project (though we should figure out the license header soon, @philips )